draft-ietf-quic-recovery.md

```diff
@@ -526,15 +524,16 @@ as a probe, unless there is no data available to send.  An endpoint MAY send up
 to two full-sized datagrams containing ack-eliciting packets, to avoid an
 expensive consecutive PTO expiration due to a single lost datagram.
 
-It is possible that the sender has no new or previously-sent data to send.  As
-an example, consider the following sequence of events: new application data is
-sent in a STREAM frame, deemed lost, then retransmitted in a new packet, and
-then the original transmission is acknowledged.  In the absence of any new
-application data, a PTO timer expiration now would find the sender with no new
-or previously-sent data to send.
+When the PTO timer expires, and there is new or previously sent data, it MUST
```

Comment:
Does "previously sent" here mean in-flight/unacknowledged, or literally any historical data?

Does "data" mean ack-eliciting frames specifically?

Reply:
I changed it to previously sent unacknowledged data, because that is the intent, though it's also a bit verbose.

Data is used fairly widely to mean ack-eliciting frames in this document.  I'm not sure if that is something we should codify in the definitions section?

Reply:
On review, maybe the existing requirement that probes be ACK-eliciting is enough. Nothing wrong with retransmitting some unacknowledged ACKs alongside your PING.